src/example.yml

```diff
@@ -3,5 +3,7 @@
   name: Project-specific roles
   become: yes
   become_method: sudo
-  roles:
-    - example
+  tasks:
+    - name: Install cisagov/example
```

Comment:
I would like a better description here because that Ansible role [does nothing](https://github.com/cisagov/skeleton-ansible-role/blob/develop/tasks/main.yml) currently.

Reply:
Please see commit ff2984a.